tests/jq.test

```diff
@@ -1580,6 +1580,15 @@ try mktime catch .
 ["a",1,2,3,4,5,6,7]
 "mktime requires parsed datetime inputs"
 
+# #2863
+[.[]|try (timegm | error) catch .]|[length,unique[]]
+[[1969],[10001],[1970,-1],[1970,12],[1970,11,0],[1970,11,32],[1970,11,30,-1],[1970,11,30,24],[1970,11,30,23,-1],[1970,11,31,23,60],[1970,11,30,23,59,-1],[1970,11,30,23,59,61],[1970,11,30,23,59,60,-1],[1970,11,30,23,59,60,7],[1970,11,30,23,59,60,4],[1970,11,30,23,59,60,4,-1],[1970,11,30,23,59,60,4,366],[1970,11,30,23,59,60,4,364,true,-43201],[1970,11,30,23,59,60,4,364,true,43201],[1970,11,30,23,59,60,4,364,true,0,true]]
```

Comment:
This tests boundary conditions.  So since we disallow years before 1969 (maybe we should allow a larger range), `[1969,...]` should fail immediately, so we have `[1969]` to test for that.